src/app/invest/invest.component.scss

```diff
@@ -152,20 +119,17 @@ p {
   }
 
   &__header-label p {
-    text-transform: uppercase;
-    color: $title-color;
-    padding-top: 8px;
-    padding-left: 10px;
-    padding-bottom: 8px;
+    padding: 8px 0 8px 10px;
   }
 
+  &__amount-label p {
+    padding: 3px 0 3px 10px;
+    font-size: $font-size-desktop-body;
+  }
+  &__header-label p,
   &__amount-label p {
     text-transform: uppercase;
     color: $title-color;
```

Comment:
When you have 2 siblings that have same style, you can apply that style like this in order to be as DRY as possible